src/pages/academy/blog/index.tsx

```diff
@@ -52,17 +52,17 @@ type ArticlesPageProps = {
 export type ArticleDataType = AllArticlesQuery['directus']['blog']
 
 const ArticlesPage = ({ data }: ArticlesPageProps) => {
-    const { is_eu_country, is_uk_country } = React.useContext(DerivStore)
+    const { is_eu, is_uk } = getCountryRule()
 
     let article_data = data.directus.blog
 
     // We need to include the !is_uk_country check together with is_eu_country because 'gb'
     // is a valid country code for both EU and UK in our country base.
-    if (is_eu_country && !is_uk_country) {
+    if (is_eu) {
```

Comment:
Can we use the `dataFilter` here?

Reply:
Sure. And moved comment in utils to make clear what exactly we doing by this function